consumer.go

```diff
@@ -440,28 +455,17 @@ func (r *Consumer) queryLookupd() {
 
 	r.log(LogLevelInfo, "querying nsqlookupd %s", endpoint)
 
-	data, err := apiRequestNegotiateV1("GET", endpoint, nil)
+	data := new(LookupResp)
```

Comment:
we avoid `new(...)`, let's use `var data LookupResp` and then `&data` below, please